WalletLibrary/WalletLibrary/Requests/Handlers/OpenIdRequestHandler.swift

```diff
@@ -38,11 +35,18 @@ struct OpenIdRequestHandler: RequestHandling {
         self.verifiedIdRequester = verifiableCredentialRequester
     }
     
+    func canHandle(rawRequest: Any) -> Bool
+    {
+        // TODO: once VC SDK logic is moved to handler and new resolver logic is implemented,
+        // reimplement with new constraints.
+        return rawRequest is any OpenIdRawRequest
+    }
+    
     /// Create a VeriifiedIdRequest based on the Open Id raw request given.
-    func handleRequest<RawRequest>(from request: RawRequest) async throws -> any VerifiedIdRequest {
+    func handle(rawRequest: Any) async throws -> any VerifiedIdRequest {
```

Comment:
Does the signature have to be changed to Any? Instead can it be the same RawRequest protocol and have the new implementation extend that protocol?